plasmapy/examples/plot_langmuir_analysis.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+"""
+Langmuir probe data analysis
+============================
+
+Let's analyze a few Langmuir probe characteristics using the
+`diagnostics.langmuir` subpackage. First we need to import the module and some
+basics.
+"""
+
+from plasmapy.diagnostics.langmuir import Characteristic, swept_probe_analysis
+import astropy.units as u
+import numpy as np
+import pickle
+import os
+
+######################################################
+# The first characteristic we analyze is a simple single-probe measurement in
+# a low (ion) temperature, low density plasma with a cylindrical probe. This
+# allows us to utilize OML theory implemented in `swept_probe_analysis`.
+# The data has been preprocessed with some smoothing, which allows us to obtain
+# a Electron Energy Distribution Function (EEDF) as well.
+
+# Load the bias and current values stored in the .p pickle file.
+path = os.path.join("langmuir_samples", "Beckers2017.p")
+bias, current = pickle.load(open(path, 'rb'))
+
+# Create the Characteristic object, taking into account the correct units
+characteristic = Characteristic(np.array(bias) * u.V,
+                                np.array(current)*1e3 * u.mA)
+
+# Calculate the cylindrical probe surface area
+probe_length = 1.145 * u.mm
+probe_diameter = 1.57 * u.mm
+probe_area = (probe_length * np.pi * probe_diameter +
```

Comment:
Huh, I've only ever seen people use either the perpendicular or parallel cross-sections of the probe area rather than both! Especially since probe orientation relative to plasma flow can have significant effects on the model. Any reason for this choice?

Perhaps we should open an issue to create a probe area function? Initially it could handle calculating the probe area for parallel, perpendicular, or total areas, then we could expand it to handle other probe geometries.

Reply:
I'm assuming a plasma without directed velocity. This should probabily be clarified.

Reply:
opened #368